internal/pkg/database/user.go

```diff
@@ -1,22 +1,24 @@
 package database
 
+import "context"
```

Comment:
The fact that we're requiring the context here indicates the context-less variants aren't used